config/initializers/cors.rb

```diff
@@ -5,12 +5,12 @@
 
 # Read more: https://github.com/cyu/rack-cors
 
-# Rails.application.config.middleware.insert_before 0, Rack::Cors do
-#   allow do
-#     origins "example.com"
-#
-#     resource "*",
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+ Rails.application.config.middleware.insert_before 0, Rack::Cors do
+   allow do
+     origins "http://localhost:3000"
+
+     resource "*",
+       headers: :any,
+       methods: [:get, :post, :put, :patch, :delete, :options, :head]
+   end
+ end
```

Comment:
- [ ] Great job working on the API but I think when I run your project I face this error because of cors so I highly recommend fixing this error to let your app work as expected without any errors. 🙌🔥

![416734244_1123582365336262_2471646214354370018_n](https://github.com/francksefu/hello/assets/58553711/233f6044-df3e-4c5c-a9f6-9519a43937e2)


- IF I tried to comment them out so it shows me like this when I want to see the messages in th frontend side

![426152825_737153558499536_7430188234606575802_n](https://github.com/francksefu/hello/assets/58553711/d4c58b2f-0ff5-46ee-88b5-8f8c4984058e)